certificate-policy.md

```diff
@@ -496,15 +496,8 @@ All Domain Validation TLS Server Authentication certificates and Organizational
 
 There is no limit on the maximum number of CT Logs which may be submitted to.  
 
-The Issuing CA shall include at least the same number and variety of SCTs in the x509v3 certificate extension for the Domain Validation TLS Server Authentication certificate or the Organizational Validation TLS Server Authentication certificate issued.
-<<<<<<< HEAD
-<<<<<<< 971515f870f94edf1a6c16f3de4605cfd0068a70
-=======
+The CA shall include at least the same number and variety of SCTs in the x509v3 certificate extension for the Domain Validation TLS Server Authentication certificate or the Organizational Validation TLS Server Authentication certificate issued.
```

Comment:
I still think the "at least the same number" is confusing because this sentence comes after "there is no limit on the maximum number of CT logs which may be submitted to"
can we move that sentence to after this one?